### PR TITLE
✨ Make two canvases into one canvas

### PR DESCRIPTION
**Pull request by DNaunin**
Now drawing everything on one canvas works and the layers are drawn in the right order. 
For next.js to not throw an error server side rendering had to be turned off (in canvas.tsx), so that the site waits until localStorage is loaded and does not only load from server side, where there is no localStorage.

### CODE REVIEW

**Review by m-wiemers — APPROVED** [review]
gj 🚀

**Comment by colefriedlaender** [comment]
Good job!

**Review by JuKahl — APPROVED** [review]
GJ 🎎

**Review by cboenke — APPROVED** [review]
LGTM 🚀

File: components/canvas/drawingarea.tsx [hunk 1]
```diff
@@ -3,94 +3,73 @@ import { useEffect, useRef, useState } from "react";
...
-    const image = new Image();
-    image.src = chosenPattern;
-    if (image.width * (x * 0.8) > 302) {
-      x = Number(x) * 0.8;
-      y = Number(y) * 0.8;
-      createPattern(chosenPattern);
+    const uploadedImage = new Image();
+    uploadedImage.src = uploadedImageSrc;
+    if (uploadedImage.width * (zoom * 0.8) > 302) {
+      setZoom(Number(zoom) * 0.8);
```
Comment (HorstFrank) [inline components/canvas/drawingarea.tsx:57]:
```suggestion
    if (uploadedImage.width * (+zoom * 0.8) > 302) {
      setZoom(+zoom * 0.8);
```
you use 'Number(zoom)' to cast 'zoom' to a Number, right? you can do it with a simple '+' ... so your 'zoom' will work in the if-statement too.

**Review by HorstFrank — APPROVED** [review]
LGFM!

**Pull request merged**